src/main/java/org/wso2/lsp4intellij/utils/FileUtils.java

```diff
@@ -286,7 +281,7 @@ public static String editorToProjectFolderPath(Editor editor) {
      * @return The uri
      */
     public static String pathToUri(@Nullable String path) {
-        return path != null ? sanitizeURI(new File(path.replace(" ", SPACE_ENCODED)).toURI().toString()) : null;
+        return path != null ? sanitizeURI(new File(path).toURI().toString()) : null;
```

Comment:
I have removed " " encoding here since there is no need for it. File.toURI() already takes care of it. 
Encoding " " with `%20` will result in duplicate encoding. 
For example: 
Original URI: `/sample/file path/abc.txt`
After Space encoding: `/sample/file%20path/abc.txt`
`File.toURI(/sample/file%20path/abc.txt)`  => `/sample/file%2520path/abc.txt`